msg/Gpgst.msg

```diff
@@ -0,0 +1,22 @@
+# Message from GPGST NMEA String
+Header header
+
+string message_id
+
+# UTC seconds from midnight
+float64 utc_seconds
+
+# Root-Mean-Squared value of standard deviation of range inputs
+float64 rms
```

Comment:
Would float32 be sufficient precision for this or any of the other fields? The examples provided by [Novatel](https://docs.novatel.com/OEM7/Content/Logs/GPGST.htm) and [Trimble](https://www.trimble.com/OEM_ReceiverHelp/V4.44/en/NMEA-0183messages_GST.html) have only 2-3 decimal places, and are close to zero, so 64 bits seems unnecessary.

Reply:
Yes, float32 should be sufficient